src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
```

Comment:
Leave a space after the previous function so that we know immediately this is related to the next function.

Also, if we're declaring global variables without wrapping the whole JavaScript file in a self inducing function, add a comment so others won't think it's declared global by mistake.